### DISCUSSION TITLE
Using it with Laravel Jetstream

### DISCUSSION THREAD

**Discussion by silicahd** [jenssegers/laravel-mongodb#2172] [Q&A]
Is anybody using this with Jetstream? I cant seem to be able to get over the session.  It seems that it create a session but it cant update it. Maybe I am doing something wrong. 

`MongoDB\Driver\Exception\BulkWriteException
E11000 duplicate key error collection: medruncrm.sessions index: id_1 dup key: { id: "4vdJsjp6JjsRIdj4xsrNxCU8B3UgU0lr3IE3jV8h" }`

**Comment by AbuobaidaMustafa** [comment]
I'm encountering the same problem

**Comment by thanhnmtech** [comment]
you can change SESSION_DRIVER=file

**Comment by Ahmed-max-b** [comment]
did you get any sollution, it seems like it doesn't work with "SESSION_DRIVER=file change", i'm facing this probleme with installation of jetstream with mongodb

**Reply by itsrexb** [reply]
you can use other drivers such memcache

**Reply by itsrexb** [reply]
You should also run $ php artisan config:cache after

**Comment by puuble** [comment]
[You can use session provider for the new driver ](https://laravel.com/docs/9.x/session#adding-custom-session-drivers)
